wordpress-mobile/WordPress-iOS
# Login Epilogue: adjust table cells for dynamic text

Ref #13413

This refactors the `BlogCell` in the `LoginEpilogue.storyboard` to allow the Site Name and URL labels to adjust with dynamic text.

To test:
- Log in.
- On the login epilogue, increase/decrease the text size.
- Verify these adjust accordingly:
  - Site Name label.
  - Site Url Label.
  - The containing cell.

| ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-03 at 19 07 43](https://user-images.githubusercontent.com/1816888/78415357-7b4c2a00-75de-11ea-9767-39b108f33ce9.png) | ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-03 at 19 07 51](https://user-images.githubusercontent.com/1816888/78415363-81420b00-75de-11ea-8286-9a88323f5422.png) |
|--------|-------|

PR submission checklist:
